js/data/locale/localematch.json

```diff
@@ -5528,7 +5528,221 @@
+        "Latn-EN-LK": "en-Latn-LK",
+        "en-MM": "en-Latn-MM",
+        "und-en-MM": "en-Latn-MM",
+        "en-EN-MM": "en-Latn-MM",
+        "Latn-EN-MM": "en-Latn-MM",
+        "en-MX": "en-Latn-MX",
+        "und-en-MX": "en-Latn-MX",
+        "en-EN-MX": "en-Latn-MX",
+        "Latn-EN-MX": "en-Latn-MX",
+        "en-MY": "en-Latn-MY",
+        "und-en-MY": "en-Latn-MY",
+        "en-EN-MY": "en-Latn-MY",
+        "Latn-EN-MY": "en-Latn-MY",
+        "en-PH": "en-Latn-PH",
+        "und-en-PH": "en-Latn-PH",
+        "en-EN-PH": "en-Latn-PH",
+        "Latn-EN-PH": "en-Latn-PH",
+        "en-PK": "en-Latn-PK",
+        "und-en-PK": "en-Latn-PK",
+        "en-EN-PK": "en-Latn-PK",
+        "Latn-EN-PK": "en-Latn-PK",
+        "en-PR": "en-Latn-PR",
+        "und-en-PR": "en-Latn-PR",
+        "en-EN-PR": "en-Latn-PR",
+        "Latn-EN-PR": "en-Latn-PR",
+        "en-RW": "en-Latn-RW",
+        "und-en-RW": "en-Latn-RW",
+        "en-EN-RW": "en-Latn-RW",
+        "Latn-EN-RW": "en-Latn-RW",
+        "en-SD": "en-Latn-SD",
+        "und-en-SD": "en-Latn-SD",
+        "en-EN-SD": "en-Latn-SD",
+        "Latn-EN-SD": "en-Latn-SD",
+        "en-TW": "en-Latn-TW",
+        "und-en-TW": "en-Latn-TW",
+        "en-EN-TW": "en-Latn-TW",
+        "Latn-EN-TW": "en-Latn-TW",
+        "en-TZ": "en-Latn-TZ",
+        "und-en-TZ": "en-Latn-TZ",
+        "en-EN-TZ": "en-Latn-TZ",
+        "Latn-EN-TZ": "en-Latn-TZ",
+        "es-CA": "es-Latn-CA",
+        "und-es-CA": "es-Latn-CA",
+        "es-ES-CA": "es-Latn-CA",
+        "Latn-ES-CA": "es-Latn-CA",
+        "es-PH": "es-Latn-PH",
+        "und-es-PH": "es-Latn-PH",
+        "es-ES-PH": "es-Latn-PH",
+        "Latn-ES-PH": "es-Latn-PH",
+        "es-US": "es-Latn-US",
+        "und-es-US": "es-Latn-US",
+        "es-ES-US": "es-Latn-US",
+        "Latn-ES-US": "es-Latn-US",
+        "und-et-EE": "et-Latn-EE",
+        "et-ET-EE": "et-Latn-EE",
+        "Latn-ET-EE": "et-Latn-EE",
+        "fr-BE": "fr-Latn-BE",
+        "und-fr-BE": "fr-Latn-BE",
+        "fr-FR-BE": "fr-Latn-BE",
+        "Latn-FR-BE": "fr-Latn-BE",
+        "fr-CH": "fr-Latn-CH",
+        "und-fr-CH": "fr-Latn-CH",
+        "fr-FR-CH": "fr-Latn-CH",
+        "Latn-FR-CH": "fr-Latn-CH",
+        "fr-GQ": "fr-Latn-GQ",
+        "und-fr-GQ": "fr-Latn-GQ",
+        "fr-FR-GQ": "fr-Latn-GQ",
+        "Latn-FR-GQ": "fr-Latn-GQ",
+        "fr-DJ": "fr-Latn-DJ",
+        "und-fr-DJ": "fr-Latn-DJ",
+        "fr-FR-DJ": "fr-Latn-DJ",
+        "Latn-FR-DJ": "fr-Latn-DJ",
+        "fr-DZ": "fr-Latn-DZ",
+        "und-fr-DZ": "fr-Latn-DZ",
+        "fr-FR-DZ": "fr-Latn-DZ",
+        "Latn-FR-DZ": "fr-Latn-DZ",
+        "fr-LB": "fr-Latn-LB",
+        "und-fr-LB": "fr-Latn-LB",
+        "fr-FR-LB": "fr-Latn-LB",
+        "Latn-FR-LB": "fr-Latn-LB",
+        "fr-RW": "fr-Latn-RW",
+        "und-fr-RW": "fr-Latn-RW",
+        "fr-FR-RW": "fr-Latn-RW",
+        "Latn-FR-RW": "fr-Latn-RW",
+        "und-ha-NG": "fr-Latn-NG",
```

Comment:
I don't think French is spoken in Nigeria at all. This should map to ha-Latn-NG instead.